app/shared/settings.py

```diff
@@ -9,32 +8,40 @@
 
 
 class Settings(BaseSettings):
```

Comment:
If the intention is to use this as a config object, I could set this up so that the property tags are setup in a way that makes them constants instead of functions. This would make it so that they could be set in all capital letters.

Reply:
indeed it is, the properties are constant all throughout